kubernetes/samples/charts/util/setup.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
```

Comment:
I don't see any reference to this script anywhere in the PR.
Is this an alternative/helper script to set up the ingress controllers besides the steps described in README files? If so, shall we describe its usage in README as well?

Reply:
Good point. Update READMEs accordingly.